src/panels/lovelace/entity-rows/hui-text-entity-row.ts

```diff
@@ -5,15 +5,18 @@ import {
   property,
   CSSResult,
   css,
+  customElement,
```

Comment:
Can you split out all these changes into a separate PR that we can merge?

Reply:
sure